examples/agent-manual/README.md

```diff
@@ -0,0 +1,44 @@
+# Lightstep Agent Manual Instrumentation Example
+
+## Build and Run
+```shell script
+export LS_ACCESS_TOKEN=your-token
+export LS_SERVICE_NAME=your-service-name
+export OTEL_EXPORTER_OTLP_SPAN_ENDPOINT=ingest.lightstep.com
+
+make buid
+make run
+```
+
+## How it works
+1. Added next dependencies:
+    ```xml
+    <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-extension-auto-annotations</artifactId>
+        <version>0.7.0</version>
+    </dependency>
+
+    <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-api</artifactId>
+        <version>0.7.0</version>
+    </dependency>
+    ```
+
+1. There are two ways to create spans:
+
+    1. Any method can be annotated with `@WithSpan` annotation to create span with duration of method call
+        ```java
+        @WithSpan
+        public void method() {
+          // ...
+        }
+        ```
+
+    1. Manually get tracer and create span
```

Comment:
is it intentional that all of the numbers here are "1." ?  Does markdown re-number for you?

Reply:
nvm I just went and looked in your branch and saw that this is what happens! 👍